docs/upgrading.md

```diff
@@ -71,10 +71,24 @@ Change the Contour image version to `docker.io/projectcontour/contour:v1.0.0-bet
 The recommended version of Envoy remains unchanged from Contour 0.15.
 Ensure the Envoy image version is `docker.io/envoyproxy/envoy:v1.11.1`.
 
+### Single pod install type now deprecated
```

Comment:
Not one hundred percent sure about this wording, but not sure how to do it better either.

Reply:
Perhaps talk about the future, not the past., eg.

### Split deployment/daemonset now the default.